tests/e2e/rudder-npm-js/src/analytics/rudder.ts

```diff
@@ -0,0 +1,109 @@
+/**
+ * This client was automatically generated by RudderTyper. ** Do Not Edit **
+ */
+
+import type { RudderAnalytics, RudderAnalyticsPreloader } from '@rudderstack/analytics-js/*';
+
+/**
+ * The analytics.js snippet should be available via window.analytics.
+ * You can install it by following instructions at: https://docs.rudderstack.com/stream-sources/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk#2-installing-the-rudderstack-javascript-sdk
+ */
+declare global {
+  interface Window {
+    rudderanalytics: RudderAnalytics | RudderAnalyticsPreloader | undefined;
+  }
+}
+
+/** The Schema object which is being used by Ajv to validate the message */
+export interface Schema {
+  $schema?: string;
+  description?: string;
+  properties?: object;
+  title?: string;
+  type?: string;
+}
+
+/** A dictionary of options. For example, enable or disable specific destinations for the call. */
+export interface Options {
+  /**
+   * Selectivly filter destinations. By default all destinations are enabled.
+   * https://docs.rudderstack.com/stream-sources/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk#4-how-to-filter-selective-destinations-to-send-event-data
```

Comment:
Please fix all these links.

Reply:
Not addressing it right now. When we will fix the type-related issues we can address this.